tripy/docs/post0_developer_guides/debugging.md

```diff
@@ -8,6 +8,7 @@ This guide outlines some methods of doing so.
 
 We include some environment variables to enable extra debugging information from MLIR-TRT:
 
+- `export TRIPY_EAGER_CACHE=1` will enable eager caching for Tripy tensors to cache all intermediate IRs for future reuse. 
```

Comment:
This is on by default right? We probably want to show how to disable it instead.

Reply:
sounds good will enable by default